.gitignore

```diff
@@ -1 +1,2 @@
 target
+.idea
```

Comment:
Development environment specific files should be excluded using global gitignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Got it. Thanks.